faqs.rst

```diff
@@ -71,16 +71,31 @@ If no output is produced, then your hwloc is not CUDA-enabled.
 
 .. _launch_large_num_jobs:
 
-How do I efficiently launch a large number of jobs to Flux?
-===========================================================
-
-See `bulksubmit.py <https://github.com/flux-framework/flux-workflow-examples/tree/master/async-bulk-job-submit>`_
-for an example workflow. You can also submit many copies of the same job using
-``flux mini submit --cc=IDSET``, or submit many jobs based on a series of
-inputs on ``stdin`` or the command line via ``flux mini bulksubmit``. See the
-`flux-mini <https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-mini.html#bulksubmit>`_
-manual page for more details.
+How do I efficiently launch a large number of jobs?
+===================================================
 
+If you have many jobs to run here are some tips to help improve efficiency
+and throughput:
+
+- Create a batch job or allocation to contain the jobs in a Flux sub-instance.
+  This improves performance over submitting them directly to the Flux system
+  instance and reduces the impact of your job on system resources and other
+  users.  See also: :ref:`batch`.
+- An obvious one, but don't wait for the first job to finish before submitting
+  the next job.  For example, replace ``flux mini run`` with
+  ``flux mini submit``.  The ``flux queue drain`` command may be used to wait
+  for all jobs to complete.  See also: :core:man1:`flux-mini`,
+  :core:man1:`flux-queue`.
```

Comment:
What about shortening to:

> The `flux mini run` command will block until the job completes. Use `flux mini submit` instead to submit multiple jobs to be scheduled and run in the background, then wait for all queued jobs to complete with `flux queue drain`. Or use the `--flags=waitable` option to `flux mini submit` and use `flux job wait --all` (this has the benefit of reporting failed jobs and exits with non-zero exit status if any job fails).

Ok, that isn't shorter, but perhaps the note about exit status is useful?

Reply:
OK, I tried to rework with your suggestions in mind and force pushed.  Could you have another look and let me know what you think?  I'm happy to iterate further if it's not quite there yet.

Reply:
Yeah that seems more complete to me, though the original wasn't bad either!